Base/res/ladybird/about-pages/settings.html

```diff
@@ -338,6 +338,23 @@ <h1>Ladybird Settings</h1>
             </div>
         </div>
 
+        <div class="card">
+            <div class="card-header">Privacy</div>
+            <div class="card-body">
+                <div class="card-group">
+                    <div class="toggle-container">
+                        <label for="do-not-track-toggle">
+                            Send web sites a "Do Not Track" request
+                        </label>
+                        <label class="toggle">
+                            <input id="do-not-track-toggle" type="checkbox" />
+                            <span class="toggle-button"></span>
```

Comment:
We actually have native support for `<input type="checkbox" switch>`, so I think you could use that instead of the `.toggle-button` implementation in `ladybird.css`.